weather-app/src/app/app.component.html

```diff
@@ -1,2 +1,5 @@
-<div class="main-container"></div>
+<div class="main-container">
+    <app-current-weather></app-current-weather>
```

Comment:
i think app-current-weather must position inside a home component

Reply:
I will import this component in next pr

Reply:
I forgot, I did it in commit [[Wa-7]: addes change in structure.